moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -541,15 +541,16 @@ double JogCalcs::decelerateForSingularity(const Eigen::VectorXd& commanded_veloc
   return velocity_scale;
 }
 
-void JogCalcs::enforceJointVelocityLimits(Eigen::VectorXd& calculated_joint_change)
+void JogCalcs::enforceJointVelocityLimits(Eigen::ArrayXd& calculated_joint_change)
```

Comment:
There should be an associated unit test to verify this math makes sense, please add

Reply:
This wasn't resolved...